### DISCUSSION TITLE
Any chance to have an update to forge 47.2.0?

### DISCUSSION THREAD

**Discussion by mibcadet** [Sinytra/ForgifiedFabricAPI#79] [Q&A] [answered]
As in title. I've build package based on 47.2.0 as it was latest for 1.20.1. However I had to drop from optifine because it was heavily conflicting many things, while moving to embeddium and lazurite I need to use forgified fabric api. And there it comes incompatible with forge 47.2.0.

Kindly asking if it is possible of course :)

**Comment by Su5eD** [comment, marked as answer]
Hi, I've released FFAPI version `0.90.7+1.10.0+1.20.1`, which adds support for running with Forge 47.2.+. From my testing, it should all work fine. If any crashes pop up, please report them to the issue tracker and I'll have a look.
